packages/wrangler/src/module-collection.ts

```diff
@@ -92,6 +102,73 @@ export default function makeModuleCollector(props: {
           modules.splice(0);
         });
 
+        // ~ start  legacy module specifier support ~
+
+        // This section detects usage of "legacy" 1.x style module specifiers
+        // and modifies them so they "work" in wrangler v2, but with a warning
+
+        const rulesMatchers = rules.flatMap((rule) => {
+          return rule.globs.map((glob) => {
+            const regex = globToRegExp(glob);
+            return {
+              regex,
+              rule,
+            };
+          });
+        });
+
+        build.onResolve(
+          {
+            filter: new RegExp(
+              [...props.wrangler1xlegacyModuleReferences.fileNames]
+                .map((fileName) => `^${fileName}$`)
```

Comment:
Is it possible for deeper paths to be imported in this way in Wrangler 1?
e.g `import HTML from "views/chat.html";`

Is it possible for modules in subdirectories to use this kind of import? And if so, should the import be relative to the current module or to the root?

Reply:
Should this regex simply match any import that does not start with a `.` and then see if it can resolve it to a real file?

Reply:
1. I checked, and it's _not_ possible for deeper paths to be here. That was part of the quirks of wrangler 1. (and why you couldn't have wasm inside npm modules, and why there was a special webpack type, etc etc) 
2. I don't want to change the regex to be any non-`.` file because then there's the whole possible clashes with node modules etc etc. But this should cover the usecases (happy to revisit if I'm wrong).